addons/web/static/src/js/views/kanban/kanban_model.js

```diff
@@ -197,11 +197,33 @@ var KanbanModel = BasicModel.extend({
      * @returns {Promise<string>} resolves to the localID of the group
      */
     loadMore: function (groupID) {
-        var group = this.localData[groupID];
-        var offset = group.loadMoreOffset + group.limit;
-        return this.reload(group.id, {
-            loadMoreOffset: offset,
-        });
+        return this.loadRecords(groupID, { loadMore: true });
```

Comment:
can we try to get rid of this (afaik, only a single use left, in accounting) ?

Reply:
More than that, no usage of this was left.